src/fsverity/ioctl.rs

```diff
@@ -35,12 +44,16 @@ pub struct FsVerityEnableArg {
 // #define FS_IOC_ENABLE_VERITY    _IOW('f', 133, struct fsverity_enable_arg)
 type FsIocEnableVerity = ioctl::WriteOpcode<b'f', 133, FsVerityEnableArg>;
 
+// ENOTTY 25 Inappropriate ioctl for device
+// XXX: should we use an errno crate instead?
+static ENOTTY: i32 = 25;
```

Comment:
Are we missing that in `rustix::io::Errno`?

https://docs.rs/rustix/latest/rustix/io/struct.Errno.html#associatedconstant.NOTTY

Reply:
Yeah, makes sense, I updated this.